packages/sdks/mitosis.config.cjs

```diff
@@ -718,6 +718,25 @@ const ANGULAR_COMPONENT_REF_UPDATE_TEMPLATE_SSR = () => ({
   },
 });
 
+/**
+ * Angular doesn't support hydration for components created dynamically.
```

Comment:
can you link to any docs or code that expands on this?

Reply:
added

Reply:
Thanks!